lib/helpers.ts

```diff
@@ -10,5 +10,5 @@ export function getDefaultOpenProjectSlashMenuItems(editor: any): any[] {
 }
 
 export const defaultOpenProjectBlockSpecs = {
-  "openProjectWorkPackage": openProjectWorkPackageBlockSpec,
+  "openProjectWorkPackage": openProjectWorkPackageBlockSpec(),
```

Comment:
When running the test project with `npm run dev` I'm getting an error:

```
Uncaught TypeError: openProjectWorkPackageBlockSpec is not a function
    at ￼helpers.ts:13:29
```

Reply:
Not sure what I did now, but for some reason the error message changed now.
```
chunk-HWZ3UXOC.js?v=70691fba:18696 Uncaught Error: [tiptap error]: The editor view is not available. Cannot access view['root']. The editor may not be mounted yet.
```

Reply:
Great catch. I had not noticed that error. I need to investigate.